frontend/src/components/MXEndpoint.js

```diff
@@ -45,6 +50,34 @@ function MXEndpoint({
     }
   }
 
+  const loadingStatusLabel = (currentStatus, statusPoint) => {
+    if (currentStatus > statusPoint) {
+      return (
+        <span className="ml-8"><CheckmarkFilled size={16} color="#2F73DA" /></span>
+        )
+    } else if (isLoading && currentStatus === statusPoint) {
+      return (
+        <span className="ml-8"><Spinner size={16} fgColor="#2F73DA" /></span>
+      )
+    }
+  }
+
+  const getNotice = () => {
+    if (jobType === 'Verification') {
+      return (
+        <span>
+          use <a href="https://docs.mx.com/api#connect_request_a_url" target="_blank" rel="noreferrer">verification mode.</a>
+        </span>
+      )
+    } else {
+      return (
+        <span>
+          <a href="https://docs.mx.com/api#connect_request_a_url" target="_blank" rel="noreferrer">include transactions.</a>
+        </span>
+      )
+    }
+  }
```

Comment:
It may be worth simplifying this a bit by returning only the message that is needed.  Then down below where you use the message, you can add in the markup for the link which is used in both returns.

```suggestion
  const getNoticeMessage = () => {
    return jobType === 'Verification' ? "use verification mode" : "include transactions"
  }
```